dashboard/lib/tasks/asset_sync.rake

```diff
@@ -14,9 +14,26 @@ namespace :assets do
     app.assets = Sprockets::Railtie.build_environment(app)
     Sprockets::Railtie.build_manifest(app).compile('application.js')
   end
+
+  desc 'Copy digested assets to non-digested file paths'
+  task no_digests: :environment do
+    assets = Dir.glob("#{dashboard_dir}/public/assets/js/**/*")
+    regex = /(-{1}[a-z0-9]{32}*\.{1}){1}/
+    assets.each do |file|
+      next if File.directory?(file) || file !~ regex
+
+      source = file.split('/')
+      source.push(source.pop.gsub(regex, '.'))
```

Comment:
You ought to be able to use [`sub`](https://ruby-doc.org/core-1.9.3/String.html#method-i-sub) instead of [`gsub`](https://ruby-doc.org/core-1.9.3/String.html#method-i-gsub) here - this replacement should only happen once per filename.